### PR TITLE
Lesson_11_Homework

### PR DESCRIPTION
**Pull request by VictorPop84**
tadam

### CODE REVIEW

File: lesson_11/pop_victor/tema11-OOP3-VictorPOP.py [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+
+    def __len__(self):
+        return len(self._students)
+
+    def __contains__(self, item):
+        for element in self._students:
+            print(str(element))
+            if item in str(element):
+                return True
+        return False
```
Comment (mariuspodean) [inline lesson_11/pop_victor/tema11-OOP3-VictorPOP.py:42]: I'm sure you can find a cleaner implementation for this ;)
It works, but there's definitely a better way

Comment (VictorPop84) [reply lesson_11/pop_victor/tema11-OOP3-VictorPOP.py:42]: the other way is commented below the code:
def __contains__(self, item):
    return item in str(self._students)

File: lesson_11/pop_victor/tema11-OOP3-VictorPOP.py [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+        # return item in str(self)
+        # return item in str(self._students)
```
Comment (VictorPop84) [inline lesson_11/pop_victor/tema11-OOP3-VictorPOP.py:44]: here

File: lesson_11/pop_victor/tema11-OOP3-VictorPOP.py (hunk 1 above)
Comment (VictorPop84) [reply lesson_11/pop_victor/tema11-OOP3-VictorPOP.py:42]: or even 
def contains(self, item):
return item in str(self)

Comment (vlad-o-git) [reply lesson_11/pop_victor/tema11-OOP3-VictorPOP.py:42]: yup

**Pull request merged**